README.md

```diff
@@ -1,4 +1,45 @@
-# Node ES Modules with TypeScript
+# Accounter
+
+Manage your taxes.
+
+## Getting started
+
+1. Switch to the correct version of Node.js:
+
+```sh
+nvm use $(cat .node-version)
+```
+
+2. Install dependencies:
+
+```sh
+yarn install
+```
+
+3. Create `.env` file:
+
+```sh
+cp .env.template .env
+```
+
+4. Run setup:
+
+```sh
+yarn local:setup
+```
+
+5. Run client and server:
+
+```sh
+yarn build
+yarn client:dev
+yarn server:dev
+```
+
+6. Visit [http://localhost:3001/](http://localhost:3001/) and sign in. The credentials to log in are
+   in your `.env` file under `AUTHORIZED_USERS`.
```

Comment:
The actual value of this env variable is json object, where attributes are user names and values are  hashed passwords (with bcrypt)

Reply:
I'll update the copy to make that more clear.